scripts/create-darwin-volume.sh

```diff
@@ -55,6 +26,19 @@ test_synthetic_conf() {
     grep -q "^nix$" /etc/synthetic.conf 2>/dev/null
 }
 
+# Create the paths defined in synthetic.conf, saving us a reboot.
+create_synthetic_objects(){
+    # Big Sur takes away the -B flag we were using and replaces it
```

Comment:
I had a feeling this might not be very stable, that's why I included the true fallback. I'd look at it as best effort so this seems fine. Perhaps even just always try both.

Reply:
I've pushed a commit to do this. :)